docs/book/getting-started/zenml-pro/saas-deployment.md

```diff
@@ -0,0 +1,196 @@
+- **Instant scaling**: Infrastructure scales with your needs
+
+### 🛡️ Security & Compliance
+- **SOC 2 Type II certified**: Enterprise-grade security controls
+- **ISO 27001 certified**: International security management standards
+- **Data sovereignty**: Your ML data stays in your infrastructure
+- **Encrypted communications**: All data in transit is encrypted
+- **Custom secret stores**: Optionally use your own secret management solution
+
+### 🚀 Production Ready from Day 1
+- **High availability**: Built-in redundancy and failover
+- **Automatic backups**: Metadata backed up continuously
+- **Monitoring included**: Health checks and alerting configured
+- **Professional support**: Direct access to ZenML experts
+
+### 👥 Collaboration Features
+- **Multi-user support**: Full team collaboration capabilities
+- **SSO integration**: Connect with your identity provider
+- **Role-based access control**: Granular permissions management
+- **Workspaces & projects**: Organize teams and resources
+
+## Ideal Use Cases
+
+ZenML Pro SaaS is perfect for:
+
+- **Startups and scale-ups** that need production MLOps quickly without infrastructure overhead
+- **Teams without dedicated DevOps** that want managed infrastructure and support
+- **Organizations with existing cloud infrastructure** comfortable with SaaS tools
+- **Teams prioritizing velocity** over complete infrastructure control
+- **POC and pilot projects** that need to demonstrate value quickly
+
+## Secret Management Options
+
+### Default: ZenML-Managed Secrets Store
+
+By default, ZenML Pro SaaS stores your cloud credentials securely in our managed secrets store. This provides:
+- Zero configuration required
+- Automatic encryption at rest and in transit
+- Access controls via RBAC
+- Audit logging of secret access
+
+### Alternative: Customer-Managed Secrets Store
+
+For organizations with strict security requirements, you can configure ZenML to use your own secrets management solution:
+- AWS Secrets Manager
+- Google Cloud Secret Manager
+- Azure Key Vault
+- HashiCorp Vault
+
+![SaaS with customer secret store](../../.gitbook/assets/cloud_architecture_saas_detailed_2.png)
+
+This keeps all credentials within your infrastructure while still benefiting from managed ZenML services - [Book a call](https://www.zenml.io/book-your-demo) with us if you want this set up.
+
+## Network Architecture
+
+### Outbound-Only Communication
+
+ZenML Pro SaaS uses outbound-only connections from your infrastructure to ZenML services:
+- No inbound connections required to your infrastructure
+- Compatible with firewall and VPN restrictions
+- Secure WebSocket connections for real-time updates
+
+### Artifact Store Access
+
+The ZenML dashboard requires read access to your artifact store to display:
+- Pipeline visualizations
+- Model comparison views
+- Artifact lineage graphs
+- Step logs and outputs
+
+You control this access by configuring appropriate cloud IAM permissions.
```

Comment:
This is misleading. The truth is this: if you give your users permission to access these things, you also implicitly give the UI permission to do so. You could say "you control who can access this information in the UI by configuring appropriate ZenML Pro RBAC permissions.

Reply:
no don't we have some setting for our wrokspace/UI to enable/disable this?